### PR TITLE
Nuke Celery

### PR DESCRIPTION
**Pull request by rauchy**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
Following #4413, we're 100% on RQ. This PR is strictly cleanup.
@jezdez - I'd love to get some context / suggestions from you regarding how to proceed with the periodic tasks. I didn't mean to be rude when I deleted them in this PR, I just wanted to remove as much as possible.

## Related Tickets & Documents
#4413 #4092

### CODE REVIEW

**Comment by jezdez** [comment]
@rauchy For sure I’ll take a look tomorrow when I’m back at the computer. I’m happy to provide code to replace the periodic tasks with RQ ones, this is a pretty important API for us. Thanks for reaching out!

**Comment by rauchy** [comment]
@jezdez great, thank you! If you don't mind, I think it might be better to submit it as a standalone PR.

File: redash/handlers/admin.py [hunk 1]
```diff
@@ -41,21 +41,6 @@ def outdated_queries():
...
-        current_user._get_current_object(),
-        {"action": "list", "object_type": "celery_tasks"},
-    )
-
-    response = {"tasks": celery_tasks()}
-
-    return json_response(response)
-
-
 @routes.route("/api/admin/queries/rq_status", methods=["GET"])
```
Comment (arikfr) [inline redash/handlers/admin.py]: Maybe we should switch the URLs not to leak the implementation detail (RQ) now that it's only RQ?

File: redash/worker.py [hunk 2]
```diff
@@ -18,9 +14,6 @@
     redis_connection,
     rq_redis_connection,
 )
-from redash.metrics import celery as celery_metrics  # noqa
```
Comment (arikfr) [inline redash/worker.py]: Need to remove `redash/metrics/celery.py` as well.

**Comment by arikfr** [comment]
@rauchy in your opinion, anything blocking us from merging this?

**Pull request merged**